src/common/utils/url.ts

```diff
@@ -113,3 +113,25 @@ const equalUrlsIgnoringSubpath = (url1: URL, url2: URL, ignoreScheme?: boolean)
 const escapeRegExp = (s: string) => {
     return s.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'); // $& means the whole matched string
 };
+
+export const isEasySSOLoginURL = (url: string): boolean => {
+    try {
+        let urlToTest = url;
+        if (!url.startsWith('http://') && !url.startsWith('https://')) {
+            urlToTest = `https://${url}`;
+        }
+        const parsed = parseURL(urlToTest);
+
+        // Check pathname and search params
+        if (parsed?.pathname === '/login/one_time_link') {
```

Comment:
This should also use `isUrlType`. We should also check for subpath on this one too.